device/cuda/src/utils/global_index.hpp

```diff
@@ -0,0 +1,21 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2025 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "traccc/device/global_index.hpp"
+
+namespace traccc::cuda::details {
+
+/// Function creating a global index in a 1D CUDA kernel
+__device__ inline device::global_index_t global_index1() {
```

Comment:
No 2 and 3 dimensional versions like for SYCL?

Reply:
🤔 It's just that those need some actual new code for them. As we don't have examples of doing this in any other place.

So it was mostly laziness on my part, not wanting to write new code plus the accompanying unit tests. 😛

Reply:
Sure, that's a valid reason in my mind. :smile: